core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -119,6 +119,24 @@
   </div>
 </div>
 
+<script>
+  if ($(window).width() < 769) {
```

Comment:
I think this will need to be rewritten using Angular somehow. You'd probably want it to function similarly for all pages of the site.

Reply:
Hi @seanlip The purpose of this PR was not to submit elegant code that could be merged with the codebase, rather give you and @tjiang11 a feel for this new UI. Will definitely change the functionality for all pages, once @tjiang11 is fine with this UI.